config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  devise_for :users
+
+  resources :categories do 
+    resources :expenses
+  end
```

Comment:
- [ ] @amon-cofie, we really need to make sure that only necessary routes are used on this project. By default, Rails creates routes for all seven of the default actions (index, show, new, create, edit, update, and destroy) for every RESTful route in your application. You can use the :only and :except options to fine-tune this behavior. The :only option specifies that only certain routes should be generated. This can really save our memory space

Reply:
Thank you for pointing that out. I will fix it 👍